sklearn/ensemble/_forest.py

```diff
@@ -1202,9 +1202,9 @@ class RandomForestRegressor(ForestRegressor):
     A random forest is a meta estimator that fits a number of classifying
     decision trees on various sub-samples of the dataset and uses averaging
     to improve the predictive accuracy and control over-fitting.
-    The sub-sample size is always the same as the original
-    input sample size but the samples are drawn with replacement if
-    `bootstrap=True` (default).
+    The sub-sample size is controlled with the max_samples parameter if
+    bootstrap=True (default), otherwise the whole dataset is used to build each
+    tree.
```

Comment:
```suggestion
    The sub-sample size is controlled with the `max_samples` parameter if
    `bootstrap=True` (default), otherwise the whole dataset is used to build each
    tree.
```